Mathlib/MeasureTheory/Measure/Lebesgue/Complex.lean

```diff
@@ -84,4 +83,38 @@ theorem volume_closedBall (a : ℂ) (r : ℝ) :
     volume (Metric.closedBall a r) = NNReal.pi * ENNReal.ofReal r ^ 2 := by
   rw [MeasureTheory.Measure.addHaar_closedBall_eq_addHaar_ball, Complex.volume_ball]
 
+section polar
+
+/-- The polar coordinates local homeomorphism in `ℂ`, mapping `r (cos θ + I * sin θ)` to `(r, θ)`.
+It is a homeomorphism between `ℂ - ℝ≤0` and `(0, +∞) × (-π, π)`. -/
+protected noncomputable def polarCoord : LocalHomeomorph ℂ (ℝ × ℝ) :=
+  equivRealProdClm.toHomeomorph.toLocalHomeomorph.trans polarCoord
+
+protected theorem polarCoord_apply (a : ℂ) :
+    Complex.polarCoord a = (Complex.abs a, Complex.arg a) := by
+  simp_rw [Complex.abs_def, Complex.normSq_apply, ← pow_two]
+  rfl
+
+protected theorem polarCoord_source :
+    Complex.polarCoord.source = {a | 0 < a.re} ∪ {a | a.im ≠ 0} := by simp [Complex.polarCoord]
+
+alias polarCoord_target := polarCoord_target
```

Comment:
I'm sure this line does what I hope it does, but that's still a lot of obfuscation, and also the name is probably not protected which could lead to clashes. Maybe better to have a true theorem
```
protected theorem polarCoord_target :
    Complex.polarCoord.target = ... := by
  ...
```
?

Reply:
Indeed, it is a bit confusing. I fixed that.